signer/Cargo.toml

```diff
@@ -66,7 +66,7 @@ hmac = { workspace = true }
 zeroize = { workspace = true }
 bip39 = { workspace = true }
 bip32 = { workspace = true, features = ["alloc", "secp256k1"], optional = true }
-schnorrkel = { workspace = true, optional = true }
+schnorrkel = { workspace = true, optional = true, features = ["getrandom"] }
```

Comment:
perhaps update the README in the signer crate that "sr25519 needs getrandom" which isn't supported on all platforms

Reply:
can we remove the getrandom feature flag now below?

https://github.com/paritytech/subxt/pull/1872/files#diff-8b295430b22e1459a5ea2f83b57cde4807b8380eac4528bfc639acb67733254cR84-R85